fast_llm/layers/transformer/config.py

```diff
@@ -473,6 +478,11 @@ class TransformerConfig(TransformerArchitectureConfig, BaseModelConfig):
     use_flash_attention: bool = Field(
         default=True, desc="Enable Flash Attention if possible.", hint=FieldHint.optional
     )
+    isolate_document_context: bool = Field(
```

Comment:
I think it was better in the BatchConfig, since it's only used for preprocessing and not by the transformer directly. Also same comment about naming.

Reply:
Let's call this `enable_cross_document_attention`, flip the boolean and make the default `True`.

Reply:
Guess that works. Or simply `cross_document_attention`